internal/db/db.go

```diff
@@ -0,0 +1,162 @@
+package db
+
+import (
+	"database/sql"
+	"errors"
+
+	"github.com/nikaydo/final/internal/model"
+	_ "modernc.org/sqlite"
+)
+
+var (
+	ErrNotFoundId = errors.New("not found")
+	ErrIncorectId = errors.New("incorrect id for updating task")
+)
+
+const (
+	schema = `CREATE TABLE IF NOT EXISTS scheduler (
+		id INTEGER PRIMARY KEY AUTOINCREMENT,
+		date CHAR(8) NOT NULL DEFAULT "",
+		title TEXT,
+		comment TEXT,
+		repeat VARCHAR(128)
+	);`
+	idx = `CREATE INDEX IF NOT EXISTS scheduler_date_index ON scheduler(date);`
+)
+
+type Database struct {
+	db *sql.DB
+}
+
+func (d *Database) InitDatabase(path string) error {
+	db, err := sql.Open("sqlite", path)
+	if err != nil {
+		return err
```

Comment:
Не ленись оборачивать, потом самому же проще будет читать ошибки
```suggestion
		return fmt.Errorf("opening database: %w", err)
```